universal_login/emails/welcome_email.json

```diff
@@ -0,0 +1,8 @@
+{
+  "template": "welcome_email",
+  "body": "<html> <body> <center> <table style='margin: 0;padding: 0; border-collapse: collapse !important;' align='center' border='0' cellpadding='0' cellspacing='0' height='100%' width='100%'> <tr> <td style='padding-left:24px'></td><td align='left' valign='top' width='480'> <img style='margin-top: 10px;' src='https://identity-public-assets-stage.s3.eu-west-1.amazonaws.com/images/wellcomecollections-150x50.png' width='150' alt='Wellcome Collection Logo'/> <hr style='border: 1px solid #edebe9; border-bottom: 0; margin: 20px 0;'/> <p style='margin-bottom: 1.3em; padding-right: 80px; font-size: 15px; color: #323130; font-family: \"Helvetica Neue\", Helvetica, Arial, sans-serif;'> Hi {{user.given_name}},</p><p style='margin-bottom: 1.3em; padding-right: 80px; font-size: 15px; color: #323130; font-family: \"Helvetica Neue\", Helvetica, Arial, sans-serif;'> Thanks for applying for a library membership. We will be in contact with you within the next 72 hours to ask for identification and confirm your membership status.</p><p style='margin-bottom: 1.3em; padding-right: 80px; font-size: 15px; color: #323130; font-family: \"Helvetica Neue\", Helvetica, Arial, sans-serif;'> In the meantime, you can access our digital collections and subscription resources.</p><![if !mso]> <p style='margin-bottom: 1.3em; display: inline-block; -webkit-font-smoothing: antialiased; font-family: \"Helvetica Neue\", Helvetica, Arial, sans-serif; font-weight: bold; font-size: 15px; appearance: button; border-radius: 6px; text-align: center; white-space: nowrap; cursor: pointer; background: #007868; border: 2px solid #007868; padding: 14px;'><a style='color: #ffffff; text-decoration: none;' href='https://wellcomecollection.org/account'>Sign into your library account</a></p><![endif]> <p style='margin-bottom: 1.3em; padding-right: 80px; font-size: 15px; color: #323130; font-family: \"Helvetica Neue\", Helvetica, Arial, sans-serif; margin-bottom: 1.3em;'> If you have any questions about your account, please email us at <a style='color: #01786c' href='{{support_url}}'>{{support_url}}</a> and we’ll be happy to help.</p><p style='margin-bottom: 1.3em; padding-right: 80px; font-size: 15px; color: #323130; font-family: \"Helvetica Neue\", Helvetica, Arial, sans-serif; margin-bottom: 1.3em;'> Thanks,<br/> Library Enquiries </p></td><td style='padding-left:24px'></td></tr></table> </center> </body></html>",
+  "from": "library@wellcomecollection.org",
+  "subject": "Welcome to Wellcome Collection",
+  "syntax": "liquid",
+  "enabled": true
+}
```

Comment:
Where does this HTML get generated? Could we have the build script pull in an HTML file that's easier to edit, and construct the JSON from there to help with dev experience?

Reply:
I've put separate html files in for the email templates